{"name":"posix::error::errno::errno_location","from_trait":false,"arguments":[],"sources":[{"loc":"file: Real(\"/home/ans5k/.cargo/registry/src/github.com-1ecc6299db9ec823/serial-0.3.4/src/posix/error.rs\") line 78","kind":{"UnsafeFnCall":"C"}}]}
